doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -2,154 +2,312 @@
 
 Using partition mode on UP2
 ###########################
-This tutorial describes the steps needed to use partition mode with the ACRN
-hypervisor on UP2.
 
-Build kernel and modules for partition mode guest
-*************************************************
+ACRN hypervisor supports partition mode, in which the User OS running in a
+privileged VM can bypass the ACRN hypervisor and access to the isolated
+PCI devices. This tutorial provides step by step instructions on how to set up
+the ACRN hypervisor partition mode on
+`UP2 <https://up-board.org/upsquared/specifications/>`_ boards running two
+privileged VMs as shown in below figure.
 
-* Build kernel and modules
+.. figure:: images/partition_mode_up2.png
+   :align: center
+   
+   Two privileged VMs running in partition mode
 
-  .. code-block:: none
+Prerequisites
+*************
 
-     git clone https://github.com/projectacrn/acrn-kernel.git
-     cd ~/acrn-kernel
-     cp kernel_config_partition_os .config
-     make oldconfig
-     make
+In this tutorial two Linux privileged VMs are started by the ACRN hypervisor.
+To set up the Linux root filesystems for each VM, follow the Clear Linux
+`bare metal installation guide <https://clearlinux.org/documentation/clear-linux/get-started/bare-metal-install#bare-metal-install>`_
+to install Clear Linux on a **SATA disk** and a **USB flash disk** prior the setup,
+as the two privileged VMs will mount the root filesystems via the SATA controller
+and the USB controller respectively.
 
-* Install modules
+Build kernel and modules for partition mode UOS
+***********************************************
 
-  First of all, please prepare one USB flash drive and one SATA disk. Plug them
-  in one PC and follow Clear Linux's `online instruction <https://clearlinux.org/documentation/clear-linux/get-started/bare-metal-install#bare-metal-install>` to install Clear Linux on them.
+#. On your development workstation, clone the ACRN kernel source tree [1]_, and
+   build the Linux kernel image that will be used to boot the privileged VMs:
 
-  Plug the USB flash drive and SATA disk into your PC. Assuming Clear Linux
-  rootfs is installed into /dev/sdx3.
-
-  .. code-block:: none
-
-     mount /dev/sdx3 /mnt/sdx3
-     cd ~/acrn-kernel
-     make INSTALL_MOD_PATH=/mnt/sdx3/ modules_install
-
-* Install kernel
-
-  First, please following Ubuntu's `online instruction <https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-desktop?_ga=2.114179015.1954550575.1530817291-1278304647.1523530035>` to install it on your UP2's eMMC.
-
-  Copy kernel image to EFI System Partition
+   .. code-block:: none
+     :emphasize-lines: 1,5,7,13,14
+
+     $ git clone https://github.com/projectacrn/acrn-kernel.git
```

Comment:
This clone command gets you the master branch, but your (very subtle) footnote says to use a particular commit ID. If you intend to have folks NOT use the master branch's code, put that information here and in a more visible way.  What commands should a user use to get the commit ID code you've tested?